misc/logging/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+use tracing_subscriber::EnvFilter;
+
+pub fn init_tracing_subscriber_with_default_env_filter() {
+    let _ = tracing_subscriber::fmt()
```

Comment:
we can just call `init_tracing_subscriber_with_env_filter` with `EnvFilter::from_default_env()` right?

Reply:
> we can just call `init_tracing_subscriber_with_env_filter` with `EnvFilter::from_default_env()` right?

yes, but `init_tracing_subscriber_with_default_env_filter` seemes neater. Can change if neede, but this way it will only take up one fn call without having to give any input and importing the `EnvFilter` to do `EnvFilter::from_default_env()`.

Reply:
what I meant is that this function, `init_tracing_subscriber_with_default_env_filter` inside can just call `init_tracing_subscriber_with_env_filter` with `EnvFilter::from_default_env()`  instead of duplicating code

Reply:
> what I meant is that this function, `init_tracing_subscriber_with_default_env_filter` inside can just call `init_tracing_subscriber_with_env_filter` with `EnvFilter::from_default_env()` instead of duplicating code

ahh, yes. done.